src/Body.jl

```diff
@@ -28,18 +28,22 @@ at time `t` using an immersion kernel of size `ϵ`.
 See Maertens & Weymouth, doi:[10.1016/j.cma.2014.09.007](https://doi.org/10.1016/j.cma.2014.09.007).
 """
 function measure!(a::Flow{N,T},body::AbstractBody;t=zero(T),ϵ=1) where {N,T}
-    a.V .= 0; a.μ₀ .= 1; a.μ₁ .= 0
+    a.V .= zero(T); a.μ₀ .= one(T); a.μ₁ .= zero(T)
```

Comment:
This is the same as `fill`, which must maintain the type. So do these actually need to be specified?

Reply:
Maybe not, but this is more correct imho.

Reply:
how can it be more correct if they compile to the same thing and waste characters? ;-)

Reply:
x)